apps/webpack.config.js

```diff
@@ -670,18 +659,58 @@ function createWebpackConfig({
               }),
             ]),
       }),
-      ...(watch
+      new PyodidePlugin(),
+      ...(envConstants.HOT
         ? [
-            new LiveReloadPlugin({
-              appendScriptTag: envConstants.AUTO_RELOAD,
-            }),
+            new webpack.HotModuleReplacementPlugin({}),
+            new ReactRefreshWebpackPlugin(),
+            // Prints a URL for accessing the Dashboard via webpack-dev-server
+            {
+              apply: compiler => {
+                compiler.hooks.afterDone.tap('PrintDashboardURL', stats => {
+                  if (stats.hasErrors()) return;
+
+                  if (!process.env.WEBPACK_SERVE) {
```

Comment:
this is set by `webpack serve`, but annoyingly, it also gets tripped in a weird way that I couldn't figure out by the concurrent task in Gruntfile.js forks. Otherwise, I would have used it more heavily (e.g. keying printing this message off WEBPACK_SERVE instead of HOT)